examples/workflow-standalone/src/di.config.ts

```diff
@@ -39,5 +41,40 @@ export default function createContainer(options: IDiagramOptions): Container {
     bindOrRebind(container, TYPES.ILogger).to(ConsoleLogger).inSingletonScope();
     bindOrRebind(container, TYPES.LogLevel).toConstantValue(LogLevel.warn);
     container.bind(TYPES.IMarqueeBehavior).toConstantValue({ entireEdge: true, entireElement: true });
+    configureInversifyLogger(container);
     return container;
 }
+
+function configureInversifyLogger(container: Container): void {
```

Comment:
Should we add that to our DI utils with a boolean flag and some default log options? I believe that might be useful for adopters as well.

Reply:
I don't think that that is a good idea since the logger functionality is provided by an additional package (inversify-logger-middleware).
If we add this to the di-util we would also need to introduce an additional (peer) dependency to `inversify-logger-middleware`.

I would rather keep this local to our standalone example. If adopters want to integrate similar logging functionality they can refer to the example as a reference.

Reply:
Ah, I see, I missed the extra dependency. Yeah, then I totally agree that we should not force that one on adopters!